docs/bit-manipulation/toggle-kth-bit.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Toggle K'th bit
+slug: /bit-manipulation/toggle-kth-bit
+---
+
+## Introduction
+
```

Comment:
In the intro section can you also add the concept of right and left shift operator. I think that would be a valuable addition. Rest seems perfect to me.

Reply:
Done